### PR TITLE
URL to operation rename

### LINKED ISSUE #44: Coverage info in report should say operations not urls
The coverage is based on operations (i.e. path + method), not urls.

### PR DESCRIPTION
**Pull request by tsiq-karold**
In test report, coverage info is labelled as url whereas it's really operation coverage.

Fixes #44

### CODE REVIEW

File: webtau-reactjs/src/report/OverallSummary.css [hunk 1]
```diff
@@ -22,7 +22,7 @@
 }
 
 .overall-http-calls-time,
-.overall-http-calls-coverage {
+.overall-operations-coverage {
```
Comment (MykolaGolubyev) [inline webtau-reactjs/src/report/OverallSummary.css]: seems like http calls is till valid here? 
need some indication what operations are as this report is also used for UI testing.

File: webtau-reactjs/src/report/OverallSummary.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ function OverallSummary({report, onSwitchToHttpCalls, onSwitchToSkippedHttpCalls
     return (
         <div className="overall-summary">
             <CallsTiming report={report} onSwitchToHttpCalls={onSwitchToHttpCalls}/>
-            <UrlCoverageSummary report={report} onSwitchToSkippedHttpCalls={onSwitchToSkippedHttpCalls}/>
+            <OperationCoverageSummary report={report} onSwitchToSkippedHttpCalls={onSwitchToSkippedHttpCalls}/>
```
Comment (MykolaGolubyev) [inline webtau-reactjs/src/report/OverallSummary.js]: same as previous css comment

File: webtau-reactjs/src/report/OverallSummary.css (hunk 1 above)
Comment (tsiq-karold) [reply webtau-reactjs/src/report/OverallSummary.css]: Yeah, I guess this one probably still make sense as http-calls, will put it back.

File: webtau-reactjs/src/report/OverallSummary.js (hunk 2 above)
Comment (tsiq-karold) [reply webtau-reactjs/src/report/OverallSummary.js]: Not sure this one is the same really.  UrlCoverageSummary presumably doesn't get used in a UI report?  Would be weird to have this be called UrlCoverageSummary when all the things it contains are "operation something".

Comment (MykolaGolubyev) [reply webtau-reactjs/src/report/OverallSummary.js]: I mean the word `operation` is too generic. Maybe `OpenApi` prefix?
In UI `operation` could be button click and coverage will be percentage of buttons clicked.

Comment (tsiq-karold) [reply webtau-reactjs/src/report/OverallSummary.js]: I see.  I misunderstood.  Will update.

**Pull request merged**